query-engine/connector-test-kit-rs/query-engine-tests/src/schemas/basic.rs

```diff
@@ -176,3 +176,7 @@ pub fn autoinc_id() -> String {
 
     schema.to_owned()
 }
+
+pub fn empty() -> String {
```

Comment:
I'd prefer just using one of the super basic schemas instead of empty. In my opinion, empty shouldn't even work because parsing and setting up the datamodel should actually not succeed. Do we surpress errors somewhere?